orderer/common/multichannel/registrar.go

```diff
@@ -687,6 +696,15 @@ func (r *Registrar) ChannelInfo(channelID string) (types.ChannelInfo, error) {
 		return info, nil
 	}
 
+	_, ok := r.pendingRemoval[channelID]
+	if ok {
+		return types.ChannelInfo{
+			Name:            channelID,
+			ClusterRelation: types.ClusterRelationNone,
```

Comment:
I am sorry... I know I said `types.ClusterRelationNone`... but now that I look at it, `types.ClusterRelationFollower` makes more sense. After all, if these were not stopped and made inactive, they would have been followers, right?

Reply:
No worries, I updated it to use Follower.